From 466b87c8e5c3a2f360794b005d8b6122aa6e648d Mon Sep 17 00:00:00 2001
From: Nan Zhong <me@nanzho.ng>
Date: Sun, 3 Dec 2023 19:46:00 -0500
Subject: [PATCH 091/188] languages: update rescript grammar (#8962)

This bump fixes a build failure of the grammer with clang.
---
 languages.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/languages.toml
+++ b/languages.toml
@@ -1523,7 +1523,7 @@ indent = { tab-width = 2, unit = "  " }
 
 [[grammar]]
 name = "rescript"
-source = { git = "https://github.com/jaredramirez/tree-sitter-rescript", rev = "65609807c628477f3b94052e7ef895885ac51c3c" }
+source = { git = "https://github.com/jaredramirez/tree-sitter-rescript", rev = "467dcf99f68c47823d7b378779a6b282d7ef9782" }
 
 [[language]]
 name = "erlang"
